src/backend/marsha/settings.py

```diff
@@ -863,6 +863,8 @@ class Development(Base):
     CACHES = {"default": {"BACKEND": "django.core.cache.backends.dummy.DummyCache"}}
     STAT_BACKEND = values.Value("marsha.core.stats.dummy_backend")
 
+    # P2P
+    P2P_LIVE_WEB_TORRENT_TRACKER_URLS = values.ListValue(["ws://localhost:8080"])
```

Comment:
I don't see where this setting is used…
Should it be `P2P_WEB_TORRENT_TRACKER_URLS` ?

Reply:
Yes I think the setting has changed before I rebase my PR on the base branch. I will change it.